### ISSUE TITLE
Home, Features, DB Schema

### ISSUE THREAD

**Issue by paulobocanegra** [ertan96/pleY#1]
# Wiki Page Home

- [x] Is the first page you see upon entering the wiki
- [x] Contains a welcome message
- [x] Contains a description sentence of the app
- [x] Contains a link/placeholder for a link to the live page
- [x] All links in the right sidebar should contain each wiki page and link to the correct page
- [x] Correctly formatted
    - [x] each wiki page is listed in bullet points
    - [x] all links route the correct page

---

# Feature List

- [x] Should have 7 Features.
    - [x] 3 of those are User Auth, Live Hosting on Render, and Production README.
    - [x] The other 4 are from the [Feature List]([https://open.appacademy.io/learn/swe-campus-hybrid/full-stack-project/Feature-checklist](https://open.appacademy.io/learn/swe-campus-hybrid/full-stack-project/mvp-checklist)) or they have clarified them with you
- [x] Includes two to three detailed bullets on functionality and presentation of feature
- [x] At least one CRUD feature, which states what CRUD operations are planned (creation, reading, updating, deletion)
- [x] Estimates how long it will take the code each Feature
- [x] Correctly formatted
    - [x] Features are listed in an ordered list
    - [x] Each Feature is broken down into bullet points

---

# Database Schema

- [x] Contains correct datatypes
- [x] Contains appropriate constraints/details (primary and foreign keys, not null, indexed, unique)
- [x] Contains bullet points after the table that state which foreign keys will reference to which table, or references to the associations which will be made
- [x] Correctly formatted
    - [x] schema is written in a table format
    - [x] table names, column names, and foreign keys are lowercased, snake_cased and `back_ticked`
    - [x] the table header column names are bolded

---

**Comment by paulobocanegra** [comment]
Features:
1,2,3 then 
4. Reviews (this is the core CRUD)
5. Map
6. Search/Filters
7. Readme

Reviews are a higher priority than Search.

**Comment by paulobocanegra** [comment]
DB Schema:
To make the development of your app easier, I would recommend not using a join table for category. It'll be easier if category is a string. Like: category: "mexican food",  if we needed more, we can comma separate them like: category: "tapas, spanish food". When we query the DB we can do: Businesses.where(category LIKE %[query]%) and it will look for any matches.

**Comment by paulobocanegra** [comment]
That being said, there is nothing wrong with making the join table. It will just be more time setting it up, seeding, and making the backend for it.

**Comment by ertan96** [comment]
You bring a great point, I have adjusted the schema to remove category table and included as a column in my business table as a string. This will make it more straightforward when just searching for the string in the search feature. Thanks for your advice! Let me know if anything else would be too complicated or needs adjusting. I have included latitude and longitude for google maps api, but haven't done extensive research on this yet to know if this is the applicable way to approach the google maps api when pinpointing later on in my project.
